waku/common/message.go

```diff
@@ -487,3 +458,32 @@ func (store *MemoryMessageStore) Pop() ([]*ReceivedMessage, error) {
 	}
 	return all, nil
 }
+
+// BytesToUintLittleEndian converts the slice to 64-bit unsigned integer.
+func BytesToUintLittleEndian(b []byte) (res uint64) {
+	mul := uint64(1)
+	for i := 0; i < len(b); i++ {
+		res += uint64(b[i]) * mul
+		mul *= 256
+	}
+	return res
+}
+
+// BytesToUintBigEndian converts the slice to 64-bit unsigned integer.
+func BytesToUintBigEndian(b []byte) (res uint64) {
+	for i := 0; i < len(b); i++ {
+		res *= 256
+		res += uint64(b[i])
+	}
+	return res
+}
+
+// ContainsOnlyZeros checks if the data contain only zeros.
+func ContainsOnlyZeros(data []byte) bool {
+	for _, b := range data {
+		if b != 0 {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
This may also be a similar case to https://github.com/status-im/status-go/pull/1947/files#r415707643 . What do you think?